src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,49 +1,70 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+interface Props {
+  selectedTodo: Todo | null;
```

Comment:
handle it in the upper component
```suggestion
  selectedTodo: Todo;
```